at/RunCukesTest.java

```diff
@@ -0,0 +1,69 @@
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereClientFactory;
+import org.fluentlenium.core.FluentAdapter;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import play.Application;
+import play.Configuration;
+import play.Environment;
+import play.Mode;
+import play.inject.guice.GuiceApplicationBuilder;
+import play.test.Helpers;
+import play.test.TestBrowser;
+import play.test.TestServer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static play.inject.Bindings.bind;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(format={"junit:target/reports/junit/junit.xml", "html:target/reports/html"})
+public class RunCukesTest {
+
+    private static Application application;
+    private static Integer port = play.api.test.Helpers.testServerPort();
+    private static TestServer testServer;
+    private static TestBrowser testBrowser;
+
+    @BeforeClass
+    public static void startPlayApp() throws Exception {
+        application = provideApplication();
+        testServer = Helpers.testServer(port, application);
+        testServer.start();
+        testBrowser = new TestBrowser(Helpers.HTMLUNIT, "http://localhost:" + port);
+    }
+
+    @AfterClass
+    public static void shutdownPlayApp() throws Exception {
+        Helpers.stop(testServer);
```

Comment:
It will start and shutdown play after each test class? If you have many tests we have experienced long execution times doing that. Would it be possible to run it once only?